account/mixins.py

```diff
@@ -1,29 +1,42 @@
+from django.http import Http404
 from django.views import View
+
+from octopoes.connector.octopoes import OctopoesAPIConnector
+from rocky.settings import OCTOPOES_API
 from tools.models import Organization, OrganizationMember
-from django.http import Http404, HttpResponseRedirect
 
 
-class OrganizationsMixin(View):
-    organization = None
-    organizationmembers = None
+def get_octopoes_api_connector(base_uri: str, organization_code: str):
+    return OctopoesAPIConnector(base_uri, organization_code)
+
+
+class OrganizationView(View):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.organization = None
+        self.octopoes_api_connector = None
```

Comment:
Would it make sense to also setup the KAT-alogus client here?

Reply:
I think it was a bad idea to put the api connector in this view and even more de KAT-alogus api connector. Not all views which has an organization needs a connector. The view which needs what it needs, calls it at the moment where it is needed.

Reply:
If the connector is cheap (eg, just an instance of an often used object) its fine to instantiate for every view. If it is expensive we could just make it a property which creates the connection once we first use it.
ideally, I think storing the current organization on the request object would make sense to keep it accessible/reachable everywhere without the need for plumping.